### ISSUE TITLE
[Request] Fail2ban plugin in Netdata

### ISSUE THREAD

**Issue by deedeefink** [TechPerplexed/Gooby#13]
Thanks for v2.

I would like the ability to view Fail2ban jail chart in Netdata. This is normally a standard plugin of Netdata which needs to be enabled/configured.

Thanks

**Comment by TechPerplexed** [comment]
No worries, do you have a link or more information on how to enable/configure this plugin in docker?

**Comment by deedeefink** [comment]
Fail2ban should work out of the box when Netdata is installed given that when you install Netdata you also install the plugins. The only condition is that the Fail2ban log/config files are readable by Netdata ([Documentation](https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/fail2ban)). By default Netdata look for log/config file at: 

log_path: '/var/log/fail2ban.log'
conf_path: '/etc/fail2ban/jail.local'

Judging on my own Gooby v2 installation var/log/fail2ban.log is in place whilst there seems to be two other critical problems:  

1. Fail2ban plugin is not installed. According to this [Plugin configuration](https://github.com/netdata/netdata/wiki/Add-more-charts-to-netdata#configuring-plugins) plugins are installed at `/usr/libexec/netdata/[plugin type]/[plugin module]`  In the case of Fail2ban the full path would be: `/usr/libexec/netdata/python.d/fail2ban.chart.py`

2. /etc/fail2ban/jail.local is not present (jail.local overrides jail.conf).  This could indicate that after Fail2ban installation a jail.local were not created with the following command: `sudo cp /etc/fail2ban/jail.conf /etc/fail2ban/jail.local`

I have no experience to manage the above / install Netdata in a Docker environment. On a different server I installed everything without Gooby (Sorry ;-) and here I used the following command to install Netdata: `bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh)` It's sort of "install everything" to ensure that I would get all the plugins and modules installed. 

That's about what I can contribute.
